### PR TITLE
Fix for Store-1280 : Revise the sort element as suggested in code review

### PR DESCRIPTION
**Pull request by tmkasun**
In this PR:

* Have changed the sort UI button and include the sorting order icon and name of the attribute used for sorting

Addresses the following issue: [STORE-1280](https://wso2.org/jira/browse/STORE-1280)

### CODE REVIEW

File: apps/publisher/modules/page-decorators.js [hunk 1]
```diff
@@ -205,13 +205,13 @@ var pageDecorators = {};
             sortObj.active = false;
             sortObj.sortNext = "+";
             sortObj.sortIcon = "sorting_asc";
-            if(sortable[i].field == sortBy){
-                if(sort == "+"){
-                    sortingListSelected.helpIcon = "sorting_asc";
+            if(sortable[i].field == sortBy) {
```
Comment (mushthaq33) [inline apps/publisher/modules/page-decorators.js]: Formatting issue

File: apps/publisher/modules/page-decorators.js [hunk 2]
```diff
@@ -205,13 +205,13 @@ var pageDecorators = {};
...
+                if(sort == "+") {
```
Comment (mushthaq33) [inline apps/publisher/modules/page-decorators.js]: Formatting issue

File: apps/publisher/modules/page-decorators.js [hunk 3]
```diff
@@ -205,13 +205,13 @@ var pageDecorators = {};
...
+                    sortingListSelected.helpIcon = "fw-sort-up margin-bottom-align";
                     sortObj.sortNext = "-";
                     sortObj.sortIcon = "sorting_asc";
-                }else if(sort == "-"){
-                    sortingListSelected.helpIcon = "sorting_desc";
+                } else if(sort == "-") {
```
Comment (mushthaq33) [inline apps/publisher/modules/page-decorators.js]: Formatting issue

File: apps/store/modules/page-decorators.js [hunk 4]
```diff
@@ -498,37 +498,36 @@ var pageDecorators = {};
...
             }
-        }else{
+        } else {
             sortBy = "createdDate";
             sort = "DES";
             sortHelp = 'Date/Time Created';
         }
-
-        if(sortBy == "overview_name" && sort == "DES"){
+        if(sortBy == "overview_name" && sort == "DES") {
```
Comment (mushthaq33) [inline apps/store/modules/page-decorators.js]: Formatting issue

File: apps/store/modules/page-decorators.js [hunk 5]
```diff
@@ -498,37 +498,36 @@ var pageDecorators = {};
...
             sortHelp = 'Name';
-            sortHelpIcon = "sorting_desc";
+            sortHelpIcon = "fw-sort-down margin-top-align";
             nameActive = true;
             nameIcon = "sorting_desc";
             nameNextSort = "ASC";
-        }else if(sortBy == "overview_name" && sort == "ASC"){
+        } else if(sortBy == "overview_name" && sort == "ASC") {
```
Comment (mushthaq33) [inline apps/store/modules/page-decorators.js]: Formatting issue

File: apps/store/modules/page-decorators.js [hunk 6]
```diff
@@ -498,37 +498,36 @@ var pageDecorators = {};
...
             sortHelp = 'Name';
-            sortHelpIcon = "sorting_asc";
+            sortHelpIcon = "fw-sort-up margin-bottom-align";
             nameActive = true;
             nameIcon = "sorting_asc";
-        }else if(sortBy == "createdDate" && sort == "DES"){
+        } else if(sortBy == "createdDate" && sort == "DES") {
```
Comment (mushthaq33) [inline apps/store/modules/page-decorators.js]: Formatting issue

File: apps/store/modules/page-decorators.js [hunk 7]
```diff
@@ -498,37 +498,36 @@ var pageDecorators = {};
...
             sortHelp = 'Date/Time Created';
-            sortHelpIcon = "sorting_desc";
+            sortHelpIcon = "fw-sort-down margin-top-align";
             dateTimeActive = true;
             dateTimeIcon = "sorting_desc";
             dateTimeNextSort = "ASC";
-        }else if(sortBy == "createdDate" && sort == "ASC"){
+        } else if(sortBy == "createdDate" && sort == "ASC") {
```
Comment (mushthaq33) [inline apps/store/modules/page-decorators.js]: Formatting issue

File: apps/store/modules/page-decorators.js [hunk 8]
```diff
@@ -498,37 +498,36 @@ var pageDecorators = {};
...
             sortHelp = 'Date/Time Created';
-            sortHelpIcon = "sorting_asc";
+            sortHelpIcon = "fw-sort-up margin-bottom-align";
             dateTimeActive = true;
             dateTimeIcon = "sorting_asc";
-        }else if(sort == "popular"){
+        } else if(sort == "popular") {
```
Comment (mushthaq33) [inline apps/store/modules/page-decorators.js]: Formatting issue

File: apps/publisher/modules/page-decorators.js (hunk 1 above)
Comment (tmkasun) [inline apps/publisher/modules/page-decorators.js]: Thanks @mushthaq33 , I have fixed it in [this commit](https://github.com/wso2/carbon-store/pull/844/commits/94c71e676e222f35e133ecddcb38d0c234247b62)

File: apps/publisher/modules/page-decorators.js (hunk 3 above)
Comment (tmkasun) [inline apps/publisher/modules/page-decorators.js]: Thanks @mushthaq33 , I have fixed it in [this commit](https://github.com/wso2/carbon-store/pull/844/commits/94c71e676e222f35e133ecddcb38d0c234247b62)

File: apps/store/modules/page-decorators.js (hunk 4 above)
Comment (tmkasun) [inline apps/store/modules/page-decorators.js]: Thanks @mushthaq33 , This code segment has been refactored and it is no longer used.

File: apps/store/modules/page-decorators.js (hunk 5 above)
Comment (tmkasun) [inline apps/store/modules/page-decorators.js]: Thanks @mushthaq33 , This code segment has been refactored and it is no longer used.

File: apps/store/modules/page-decorators.js (hunk 7 above)
Comment (tmkasun) [inline apps/store/modules/page-decorators.js]: Thanks @mushthaq33 , This code segment has been refactored and it is no longer used.

File: apps/store/modules/page-decorators.js (hunk 6 above)
Comment (tmkasun) [inline apps/store/modules/page-decorators.js]: Thanks @mushthaq33 , This code segment has been refactored and it is no longer used.

File: apps/publisher/modules/page-decorators.js (hunk 2 above)
Comment (tmkasun) [inline apps/publisher/modules/page-decorators.js]: Thanks @mushthaq33 , This code segment has been refactored and it is no longer used.

**Pull request merged**